### DISCUSSION TITLE
Minimum ansible version needed

### DISCUSSION THREAD

**Discussion by aminvakil** [ansible/ansible-lint#1094] [General] [answered]
##### Summary
State the minimum ansible version which is needed

##### Issue Type

- Feature Idea

##### Additional Information
It can be used for ansible-galaxy roles for example.

**Comment by ssbarnea** [comment, marked as answer]
That is clearly stated in the first sentence of the project README....

Please do not use the issue tracker for support. We have discussions for that.

**Reply by aminvakil** [reply]
Sorry, I wasn't clear enough, I meant this as a feature, run ansible-lint on a role and know the minimum ansible version which is needed to play that role.

I have asked this on IRC and you told me to open an issue about it.

**Reply by ssbarnea** [reply]
No problem, I created https://github.com/ansible/ansible-lint/issues/1095
